ql/src/java/org/apache/hadoop/hive/ql/plan/MapWork.java

```diff
@@ -934,4 +938,20 @@ public MapExplainVectorization getMapExplainVectorization() {
     }
     return new MapExplainVectorization(this);
   }
+
+  public List<Path> getInputPaths() {
+    return inputPaths;
+  }
+
+  public void setInputPaths(List<Path> inputPaths) {
+    this.inputPaths = inputPaths;
+  }
+
+  public void setUseInputPathsDirectly(boolean useInputPathsDirectly) {
+    this.useInputPathsDirectly = useInputPathsDirectly;
+  }
+
+  public boolean useInputPathsDirectly() {
```

Comment:
please rename to `isUseInputPathsDirectly`

Reply:
Renamed.